.buildkite/common.py

```diff
@@ -297,8 +297,8 @@ def _adapt_group(self, group):
         for step in group["steps"]:
             step["command"] = prepend + step["command"]
             if self.shared_build is not None:
-                step["depends_on"] = (
-                    "build_" + DEFAULT_INSTANCES[step["agents"]["instance"]]
+                step["depends_on"] = self.build_key(
+                    DEFAULT_INSTANCES[step["agents"]["instance"]]
```

Comment:
It seems the changes from `key_prefix` to `set_key` is mixed into the commit "buildkite: fix agents passed as a list instead of dict"

Reply:
Yep, just noticed it too. Fixing.